### ISSUE TITLE
Felwood Level exploit

### ISSUE THREAD

**Issue by siscostrife** [chromiecraft/chromiecraft#908]
Salve via Disenchanting Quest ID 5886
 quest is repeatable for 3000 exp and can be abused to level from 49-60 within a couple minutes

**Comment by Stump876** [comment]
Please don't delete the template.

**Comment by Azcobu** [comment]
Sorry about tags, linked the Druid T1 issue by mistake.

Had a look at this one with a lvl 49 lock with enchanting. The quest referred to gave 6K XP on the first two handins, which is the amount listed on https://wotlkdb.com/?quest=5886, but I didn't get any XP for any handins after that. 

Screenshot showing no XP gains listed for any handins.
![WoWScrnShot_062121_010821](https://user-images.githubusercontent.com/81782124/122680259-7304fc80-d22d-11eb-8487-c750bb7b46b9.jpg)

I also tried relogging and resetting my AC server on the off-chance that might reset the questgiver, but it had no effect as expected. Also tried handing in the Salve Via Hunting quest and alternating, but that didn't do anything either. So end result, I got 12.2K xp, but did not get to lvl 60 in a couple of minutes. I'm paranoid enough to wonder if there's some other way to reset a repeatable questgiver's rewards, but if so, didn't spot it.
